tests/metrics/test_jsd.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from sdgx.metrics.column.jsd import JSD
+
+# 创建测试数据
+real_data_discrete = pd.DataFrame(
+    {
+        "col1": ["a", "b", "c", "d", "e"],
+        "col2": ["a", "b", "b", "b", "e"],
+    }
+)
+
+synthetic_data_discrete = pd.DataFrame(
+    {
+        "col1": ["a", "c", "d", "b", "b"],
+        "col2": ["a", "c", "a", "a", "e"],
+    }
+)
+
+real_data_cotinuous = pd.DataFrame(
+    {
+        "col1": [1, 1, 2, 2, 3],
+        "col2": [4, 4, 5, 5, 6],
+    }
+)
+
+synthetic_data_cotinuous = pd.DataFrame(
+    {
+        "col1": [1, 2, 2, 3, 3],
+        "col2": [4, 5, 5, 6, 6],
+    }
+)
+
+# 创建 JSD 实例
+jsd_instance = JSD()
+
```

Comment:
Can we make this as a `pytest.fixture` or parametrize?

Reply:
@sjh120  `pytest.fixture` should be OK, please refer to other test case in `test/`, such as `tests/models/test_ctgan.py`.